src/coord/src/session/vars.rs

```diff
@@ -112,17 +114,22 @@ const QGM_OPTIMIZATIONS: ServerVar<bool> = ServerVar {
     description: "Enables optimizations based on a Query Graph Model (QGM) query representation.",
 };
 
-const SEARCH_PATH: ServerVar<[&str]> = ServerVar {
-    name: static_uncased_str!("search_path"),
-    value: &[
-        MZ_CATALOG_SCHEMA,
-        PG_CATALOG_SCHEMA,
-        DEFAULT_SCHEMA,
-        MZ_TEMP_SCHEMA,
-    ],
-    description:
-        "Sets the schema search order for names that are not schema-qualified (PostgreSQL).",
-};
+lazy_static! {
+    static ref DEFAULT_SEARCH_PATH: [String; 4] = {
+        [
+            MZ_CATALOG_SCHEMA.to_owned(),
+            PG_CATALOG_SCHEMA.to_owned(),
```

Comment:
Since these are implicitly added to every search path, we can probably take them out of the default search path. This would also more closely match Postgres, which don't include these in `SHOW SEARCH_PATH` by default.